### DISCUSSION TITLE
Simulated motor container

### DISCUSSION THREAD

**Discussion by 8ryn** [epics-containers/epics-containers.github.io#36] [Q&A] [answered]
I was wondering if there is any container which simulates a motor which could interface with the ioc-pmac container for a simulated setup? 
My initial plan was to create a very straightforward python container and run the simulated PMAC at https://github.com/dls-controls/pmac/blob/dls-master/sim/SimPMAC.py but I'm running into an issue connecting that to an IOC so I thought it was worth checking if there was a pre-existing container?

**Comment by 8ryn** [comment, marked as answer]
The issue I was previously facing simply using Diamond's SimPMAC.py has now been resolved, it was simply a case of the HOST of the PMACServer being set to localhost and hence not interacting with the Docker host.

**Reply by gilesknap** [reply]
That was a quick answer. Thanks for letting us know that SimPMAC works in a container.
